splunk_test.go

```diff
@@ -1411,7 +1412,7 @@ func TestCannotSendAfterClose(t *testing.T) {
 		t.Fatal("Driver should not allow to send messages after close")
 	}
 
-	if len(hec.messages) != 1 {
+	if !(len(hec.messages) > 0) {
```

Comment:
What is the expected value here? If its > "0", then please update the error message accordingly.